src/azure-cli/azure/cli/command_modules/storage/tests/latest/test_storage_azcopy_scenarios.py

```diff
@@ -11,6 +11,7 @@
 
 class StorageAzcopyTests(StorageScenarioMixin, LiveScenarioTest):
 
+    test_resources_count = 0
```

Comment:
It shouldn't need it any more. There is a fix for it already. You can pull the latest change and sync.